.github/workflows/docker.yml

```diff
@@ -1,4 +1,4 @@
-name: Build, test, and push Docker Images
+name: Build, merge tags, update wiki, test recipes
 
 # [FAST_BUILD] in the PR title makes this workflow only build
```

Comment:
We need to move comment about `[FAST_BUILD]` to a new file.

Reply:
Reverted to the original workflow, so let's do this in another PR.